sql/sys_vars.cc

```diff
@@ -6683,3 +6683,28 @@ static Sys_var_ulonglong Sys_max_rowid_filter_size(
        SESSION_VAR(max_rowid_filter_size), CMD_LINE(REQUIRED_ARG),
        VALID_RANGE(1024, (ulonglong)~(intptr)0), DEFAULT(128*1024),
        BLOCK_SIZE(1));
+
+static Sys_var_mybool Sys_redirect_enabled(
+       "redirect_enabled",
+       "Enable return redirection endpoint",
+       GLOBAL_VAR(redirect_enabled), CMD_LINE(OPT_ARG),
+       DEFAULT(FALSE), NO_MUTEX_GUARD, NOT_IN_BINLOG, ON_CHECK(0),
+       ON_UPDATE(0));
+
+static Sys_var_charptr Sys_redirect_server_host(
+       "redirect_server_host",
+       "The server host that client can choose to redirect to.",
+       READ_ONLY GLOBAL_VAR(redirect_server_host), CMD_LINE(OPT_ARG),
+       DEFAULT("localhost"));
+
+static Sys_var_charptr Sys_redirect_server_port(
+       "redirect_server_port",
+       "The server host port that client can choose to redirect to.",
+       READ_ONLY GLOBAL_VAR(redirect_server_port), CMD_LINE(OPT_ARG),
+       DEFAULT("3306"));
+
+static Sys_var_charptr Sys_redirect_server_ttl(
+       "redirect_server_ttl",
+       "The ttl (time to live) in seconds the redirected endpoint can be trusted.",
```

Comment:
The descriptions of these variables could be clarified a bit. It's a bit hard for a non-native reader like me to comprehend what this actually means.

Reply:
Working on clarification. It will be of great help if you would like to specify which part of the description is confusing to you, if I'm not asking too much...

Reply:
It might be just me. Perhaps the documentation for these variables will explain it better once it is added.

Reply:
Clarified description. Any suggestions on further refining these variable description are most welcome.